libs/viewer/src/SimpleViewer.cpp

```diff
@@ -1450,6 +1451,34 @@ void SimpleViewer::updateUserInterface() {
         if (ImGui::CollapsingHeader("Indirect light")) {
             ImGui::SliderFloat("IBL intensity", &light.iblIntensity, 0.0f, 100000.0f);
             ImGui::SliderAngle("IBL rotation", &light.iblRotation);
+
+            if (ImGui::RadioButton("Infinite", iblOptions.iblTechnique == 0)) {
+                iblOptions.iblTechnique = 0;
+            }
+            ImGui::SameLine();
+            if (ImGui::RadioButton("Sphere", iblOptions.iblTechnique == 1)) {
+                iblOptions.iblTechnique = 1;
+            }
+            ImGui::SameLine();
+            if (ImGui::RadioButton("Box", iblOptions.iblTechnique == 2)) {
+                iblOptions.iblTechnique = 2;
+            }
+
+            if (iblOptions.iblTechnique == 1) {
+                static float radius = std::sqrt(iblOptions.iblHalfExtents.x);
+                ImGui::SliderFloat("Sphere radius", &radius, 0.0f, 256.0f);
+                iblOptions.iblHalfExtents.x = radius * radius;
+            }
+            else if (iblOptions.iblTechnique == 2) {
+                static filament::math::float3 iblCenter = iblOptions.iblCenter;
+                static filament::math::float3 iblHalfExtents = iblOptions.iblHalfExtents;
+
+                ImGui::SliderFloat3("Box center", iblCenter.v, -10.0f, 10.0f);
```

Comment:
`iblCenter` also affects the sphere (`iblTechnique == 1`), so these sliders should also be exposed there, right?

Reply:
Fixed in https://github.com/shapr3d/filament/pull/44/commits/a34287c2f26493e148e21891845547aa43b92757